### PR TITLE
fix(content-details)

### PR DESCRIPTION
**Pull request by ecovirtual**
Changes:
* Update text description for each step

### CODE REVIEW

File: packages/nextjs/pages/age-restricted-example/GenerateProof.tsx [hunk 1]
```diff
@@ -94,38 +94,52 @@ export const GenerateProof = ({ requiredBirthYear }: { requiredBirthYear: number
...
+            <li>
+              2. The proof generation should happen in an environment trusted by<strong>Alice</strong> (ex: locally in
+              her laptop or phone)
+            </li>
+          </ol>
         </p>
         <p>
-          The circuit is located in{" "}
+          A circuit written in <strong>Noir</strong> is used for generating the proof and for generating a
+          Solidity-verifier. The <strong>Ballon Store</strong> will use the proof as input to execute the verifier
```
Comment (FilipHarald) [inline packages/nextjs/pages/age-restricted-example/GenerateProof.tsx:111]:
```suggestion
          proof-verifier in Solidity. The <strong>Ballon Store</strong> will use the proof as input to execute the verifier
```

File: packages/nextjs/pages/age-restricted-example/GenerateProof.tsx [hunk 2]
```diff
@@ -94,38 +94,52 @@ export const GenerateProof = ({ requiredBirthYear }: { requiredBirthYear: number
...
+          The proof is generated in the browser using the following libraries:
+          <ul>
+            <li>
+              * <CodeText text="aztec/bb.js" />
+            </li>
+            <li>
+              * <CodeText text="noir-lang/acvm_js" />
+            </li>
+          </ul>
+          Check the implementation here:{" "}
```
Comment (FilipHarald) [inline packages/nextjs/pages/age-restricted-example/GenerateProof.tsx:127]:
```suggestion
          Check out the implementation here:{" "}
```

File: packages/nextjs/pages/age-restricted-example/GenerateProof.tsx [hunk 3]
```diff
@@ -94,38 +94,52 @@ export const GenerateProof = ({ requiredBirthYear }: { requiredBirthYear: number
...
           <a href="https://github.com/Kryha/scaffold-eth-2-noir/blob/main/packages/nextjs/utils/noir/noirBrowser.ts">
             <CodeText text="packages/nextjs/utils/noir/noirBrowser.ts" />
           </a>
-          . In this file you can see that the proof is generated by using the imported libraries{" "}
-          <CodeText text="aztec/bb.js" /> and <CodeText text="noir-lang/acvm_js" />. The proof can also be generated
-          locally, by calling <CodeText text="nargo prove" />. The predefined circuit ABI code used to generate the
-          proof can be found in{" "}
+        </p>
+        <p>
+          The proof can also be generated locally (outside of a browser), by calling <CodeText text="nargo prove" />.
```
Comment (FilipHarald) [inline packages/nextjs/pages/age-restricted-example/GenerateProof.tsx:133]:
```suggestion
          The proof can also be generated locally (outside of a browser), by calling <CodeText text="nargo prove" />. <CodeText text="nargo" /> is the CLI-tool for Noir.
```

File: packages/nextjs/pages/age-restricted-example/GenerateProof.tsx [hunk 4]
```diff
@@ -94,38 +94,52 @@ export const GenerateProof = ({ requiredBirthYear }: { requiredBirthYear: number
...
+        </p>
+        <p>
+          The predefined circuit ABI code used to generate the proof can be found in{" "}
           <a href="https://github.com/Kryha/scaffold-eth-2-noir/blob/main/packages/nextjs/generated/circuits.json">
             <CodeText text="packages/nextjs/generated/circuits.json" />
           </a>
           , but we could recompile it using <CodeText text="nargo compile" />. <CodeText text="nargo" /> is the CLI-tool
```
Comment (FilipHarald) [inline packages/nextjs/pages/age-restricted-example/GenerateProof.tsx:140]:
```suggestion
          , but we could recompile it using <CodeText text="nargo compile" />.
```

File: packages/nextjs/pages/age-restricted-example/GenerateProof.tsx [hunk 5]
```diff
@@ -94,38 +94,52 @@ export const GenerateProof = ({ requiredBirthYear }: { requiredBirthYear: number
...
           for Noir.
```
Comment (FilipHarald) [inline packages/nextjs/pages/age-restricted-example/GenerateProof.tsx:141]:
```suggestion
```

File: packages/nextjs/pages/age-restricted-example/GenerateProof.tsx [hunk 6]
```diff
@@ -94,38 +94,52 @@ export const GenerateProof = ({ requiredBirthYear }: { requiredBirthYear: number
...
           </a>
-          . It&apos;s written using Noir. It is the Noir-circuit that is used both for generating the proof and for
-          generating a Solidity-verifier that can help the balloon store verify the proof on-chain.
         </p>
         <p>
-          The proof is generated in the browser from the file{" "}
+          The proof is generated in the browser using the following libraries:
+          <ul>
+            <li>
+              * <CodeText text="aztec/bb.js" />
```
Comment (FilipHarald) [inline packages/nextjs/pages/age-restricted-example/GenerateProof.tsx:121]:
```suggestion
              - <CodeText text="aztec/bb.js" />
```

So we don't mix it with the other asterix.

File: packages/nextjs/pages/age-restricted-example/GenerateProof.tsx [hunk 7]
```diff
@@ -94,38 +94,52 @@ export const GenerateProof = ({ requiredBirthYear }: { requiredBirthYear: number
...
+            </li>
+            <li>
+              * <CodeText text="noir-lang/acvm_js" />
```
Comment (FilipHarald) [inline packages/nextjs/pages/age-restricted-example/GenerateProof.tsx:124]:
```suggestion
              - <CodeText text="noir-lang/acvm_js" />
```

File: packages/nextjs/pages/age-restricted-example/BirthDateSignature.tsx [hunk 8]
```diff
@@ -64,16 +64,16 @@ export const BirthDateSignature = ({ aliceDefaultAge }: { aliceDefaultAge: numbe
...
+          <strong>Alice</strong> and the <strong>Balloon Store</strong> need to agree on who to trust in order to
+          certify Alice age. In this example the <strong>Town Hall</strong> acts as a trusted third party, but on a
+          different scheme the certifying entity could be completely different. The only requirement is that both Alice
+          & the Ballon Store trust the signature (certification) provided by this third party.
         </p>
         <p>
-          When the balloon store implemented their zero knowledge proof solution they made sure that they are using the
-          same format as the Town Hall for constructing the claim that is being signed. In this project the claim
-          construction for the Town Hall can be found in{" "}
+          The Ballon Store has previously decide to use the same claim format as the Town Hall, to facilitate the zk
```
[the remark this answers is not in the document]
Comment (ecovirtual) [reply packages/nextjs/pages/age-restricted-example/BirthDateSignature.tsx:75]: Good point. I'll update it.

File: packages/nextjs/pages/age-restricted-example/BirthDateSignature.tsx [hunk 9]
```diff
@@ -64,16 +64,16 @@ export const BirthDateSignature = ({ aliceDefaultAge }: { aliceDefaultAge: numbe
...
+          <strong>Alice</strong> and the <strong>Balloon Store</strong> need to agree on who to trust in order to
+          certify Alice age. In this example the <strong>Town Hall</strong> acts as a trusted third party, but in a
+          different set-up the certifying entity could be completely different. The only requirement is that both Alice
+          & the Ballon Store trust the signature (certification) provided by this third party.
         </p>
         <p>
-          When the balloon store implemented their zero knowledge proof solution they made sure that they are using the
-          same format as the Town Hall for constructing the claim that is being signed. In this project the claim
-          construction for the Town Hall can be found in{" "}
+          The Ballon Store has implemented the same claim format as the Town Hall, this enables a more straght forward
```
Comment (FilipHarald) [inline packages/nextjs/pages/age-restricted-example/BirthDateSignature.tsx:75]:
```suggestion
          The Ballon Store has implemented the same claim format as the Town Hall, this enables
```

File: packages/nextjs/pages/age-restricted-example/BirthDateSignature.tsx [hunk 10]
```diff
@@ -64,16 +64,16 @@ export const BirthDateSignature = ({ aliceDefaultAge }: { aliceDefaultAge: numbe
...
     <div className="grid grid-cols-2 gap-6 max-w-7xl">
       <div>
-        <h1 className="text-3xl font-bold">Step 1: Town Hall 🏛 generates the signature 📜</h1>
+        <h1 className="text-3xl font-bold">Step 1: Town Hall 🏛 signs 📜 the birth date</h1>
         <p>
-          Alice recognizes that, in order for her to not have to share her age with the balloon store, she at least has
-          to share her age with a trusted third party. In this case, the balloon store has selected the Town Hall to be
-          the trusted third party🏛. Alice accepts the fact that she has to share her age with the Town Hall.
+          <strong>Alice</strong> and the <strong>Balloon Store</strong> need to agree on who to trust in order to
+          certify Alice age. In this example the <strong>Town Hall</strong> acts as a trusted third party, but on a
```
[the remark this answers is not in the document]
Comment (ecovirtual) [reply packages/nextjs/pages/age-restricted-example/BirthDateSignature.tsx]: I think this is already updated in the last commit.

**Pull request merged**